src/main/java/org/cyclonedx/gradle/CycloneDxTask.java

```diff
@@ -135,6 +134,7 @@ public CycloneDxTask() {
         componentVersion.convention(getProject().getVersion().toString());
 
         includeConfigs = getProject().getObjects().listProperty(String.class);
+        includeConfigs.addAll("runtimeClasspath", "compileClasspath", "annotationProcessor");
```

Comment:
I think it should all configurations, but test ones. Otherwise it skips not only tests.

Reply:
Not sure I understand what you mean. 

I agree that it is an breaking change.
The issue was that the plugin contains configs that are not included in the maven plugin. https://github.com/CycloneDX/cyclonedx-gradle-plugin/issues/416#issuecomment-2071903464

Reply:
The problem is that you cannot fully match maven plugin behavior just because it's a different build system. Using Gradle, you can custom configurations and use them during runtime. Using proposed changed to the default config, plugin will skip it which is incorrect.